packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -27,7 +29,15 @@ const styles = {
 };
 
 const GlobalHeader = ({ editUrl, className, search }) => {
-  const [isOverlayOpen, setIsOverlayOpen] = useState(false);
+  const location = useLocation();
+  const [isOverlayOpen, setIsOverlayOpen] = useState(
+    Boolean(qs.parse(location.search).q)
```

Comment:
Does this return `true` or `false` if the query param is `q=` (no search term)? Since empty string is falsey, I suspect this will initially be set to `false`.